### PR TITLE
use GitHub actions instead of travis and upload code coverage

### LINKED ISSUE #4: move ci to github workflows
travis-ci.com is not building it and I can't even find a way to make it do so from the web interface.

### PR DESCRIPTION
**Pull request by terencehonles**
fixes: #4

Hopefully the additional changes are OK, but figured I'd help out here since I'm pretty familiar with GitHub Actions.

### CODE REVIEW

**Comment by terencehonles** [comment]
You can see the run against my fork https://github.com/terencehonles/pyuwsgimemhog/actions/runs/778547439

File: setup.cfg [hunk 1]
```diff
@@ -14,10 +14,6 @@ replace = __version__ = '{new_version}'
 [bdist_wheel]
 universal = 1
 
-[aliases]
-# Define setup.py command aliases here
-test = pytest
-
 [tool:pytest]
-collect_ignore = ['setup.py']
```
Comment (terencehonles) [inline setup.cfg:22]: This was warning about an unknown option: https://github.com/terencehonles/pyuwsgimemhog/runs/2421912545?check_suite_focus=true#step:6:36 and the change to use `addopts` is how I've used flags in the past.

File: setup.py [hunk 2]
```diff
@@ -13,8 +13,6 @@
 
 requirements = ['Click>=6.0', ]
 
-setup_requirements = ['pytest-runner', ]
```
Comment (terencehonles) [inline setup.py:16]: pytest-runner suggests it's deprecated and one should use tox (which you already are) https://github.com/pytest-dev/pytest-runner/#deprecation-notice

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade tox tox-gh-actions
+
+      - name: Tox tests
+        run: |
+          tox -- --cov=pyuwsgimemhog --cov-report xml
+
+      - name: Upload coverage
```
Comment (xrmx) [inline .github/workflows/ci.yml:46]: please drop codecov upload and just print the textual representation on the console output

**Comment by xrmx** [comment]
Thanks a lot! I've applied the requested changes in #8

**Comment by terencehonles** [comment]
@xrmx thanks for the release! Gonna start using this soon. I've had a chance to test out the Django example with production data for about a day so hopefully we find this useful!

Definitely better than staring at `top` and I hadn't enabled memory reporting before so it's nice to know the memory is reported in the logs and can be attributed to individual requests 🙂.

**Pull request closed without merging**